subprojects/execution/src/main/java/org/gradle/internal/execution/steps/ResolveChangesStep.java

```diff
@@ -106,10 +106,6 @@ private static IncrementalInputProperties createIncrementalInputProperties(UnitO
         switch (inputChangeTrackingStrategy) {
             case NONE:
                 return IncrementalInputProperties.NONE;
-            //noinspection deprecation
-            case ALL_PARAMETERS:
-                // When using IncrementalTaskInputs, keep the old behaviour of all file inputs being incremental
-                return IncrementalInputProperties.ALL;
```

Comment:
`IncrementalInputProperties.ALL` should also be removed.

Reply:
Removed